packages/backend/common/helpers/mailingOrderMessage.ts

```diff
@@ -0,0 +1,25 @@
+import { telegram } from 'App/Services/Telegram'
+import Bnovo from 'App/Services/Bnovo'
+import { Order } from 'App/Models/Order'
+import { DateTime } from 'luxon'
+
+export async function mailingOrderMessage (order: InstanceType<Order>): Promise<void> {
+  const { dateFrom, dateTo, apartmentId, fixedPrice, name, email, phone, prices } = order
+  const surname = name.split(' ')[0]
+  const payload = { name, surname, email, phone }
+
+  await telegram.createMessageBody(dateFrom, dateTo, apartmentId, fixedPrice, name, email, phone)
+
+  class NewBnova extends Bnovo {
```

Comment:
```suggestion
  class NewBnovo extends Bnovo {
```

Но вообще так переопределять классы внутри хелперов не очень хорошая практика.